<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ULTP_284" name="ExamTaker-VerifyErrorMsg3_CreateExamTaker" tests="1" failures="0" timestamp="9 Sep 2015 10:19:31 GMT" time="0.016" errors="0">
  <testcase name="@BeforeTest checkTestSkip" time="0.0" classname="com.SuiteExamTaker.TestCase_VerifyErrorMsg3_CreateExamTaker">
    <skipped/>
  </testcase> <!-- @BeforeTest checkTestSkip -->
  <testcase name="@AfterMethod DefaultResult" time="0.0" classname="com.SuiteExamTaker.TestCase_VerifyErrorMsg3_CreateExamTaker">
    <skipped/>
  </testcase> <!-- @AfterMethod DefaultResult -->
  <testcase name="CreateNewETwithoutStudentID" time="0.012" classname="com.SuiteExamTaker.TestCase_VerifyErrorMsg3_CreateExamTaker">
    <skipped/>
  </testcase> <!-- CreateNewETwithoutStudentID -->
</testsuite> <!-- ExamTaker-VerifyErrorMsg3_CreateExamTaker -->
